### ISSUE TITLE
Conduct accessibility review

### ISSUE THREAD

**Issue by trullock** [MyUNiDAYS/competency-framework#40]
Create issues for each a11y failing found

**Comment by chiangs** [comment]
I'll do a few of these :)
